### PR TITLE
Workspaces

### PR DESCRIPTION
**Pull request by AlexVipond**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -12,12 +12,12 @@
   * [Add a contributor](getting-started/first-steps.md#add-a-contributor)
 * [Quick tips](getting-started/quick-tips.md)
 * [Webinars](getting-started/webinars.md)
-* [FAQ](getting-started/faq.md)
```
Comment (rymohr) [inline SUMMARY.md:15]: @AlexVipond why'd you drop the FAQ here?

File: SUMMARY.md [hunk 2]
```diff
@@ -144,8 +144,6 @@
 * [How do I copy a map to another project](faq/how-do-i-copy-a-map-to-another-project.md)
 * [How do I control label size while scaling element size?](faq/how-do-i-control-label-size-while-scaling-element-size.md)
 * [How do I control text wrapping?](faq/how-do-i-control-text-wrapping.md)
-* [How do I convert a project from private to public?](faq/how-do-i-convert-a-project-from-private-to-public.md)
```
Comment (rymohr) [inline SUMMARY.md:147]: I think we should keep the individual FAQs even if the information is consolidated in other guides. By removing them from the index we just make it harder for people to find the answers they're looking for.

File: guides/organizations.md [hunk 3]
```diff
@@ -1,97 +0,0 @@
-# Organizations
```
Comment (rymohr) [inline guides/organizations.md:1]: Should we have this guide redirect instead of deleting it?

File: SUMMARY.md (hunk 1 above)
Comment (AlexVipond) [reply SUMMARY.md:15]: Deleting this one as the questions inside it didn't feel like they were frequently asked, and the ones that were compelling are already answered in more searchable articles in the FAQ section

File: guides/organizations.md (hunk 3 above)
Comment (AlexVipond) [reply guides/organizations.md:1]: Good catch, intended to do this but forgot!

File: SUMMARY.md (hunk 2 above)
Comment (AlexVipond) [reply SUMMARY.md:147]: Good call, I'll restore sometime today

**Pull request merged**